app/src/main/java/com/sun/basic_japanese/alphabet/allpages/AlphabetFragment.kt

```diff
@@ -0,0 +1,66 @@
+package com.sun.basic_japanese.alphabet.allpages
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.sun.basic_japanese.R
+import com.sun.basic_japanese.alphabet.eachpage.AlphabetPageFragment
+import com.sun.basic_japanese.alphabet.adapter.PagerAdapter
+import com.sun.basic_japanese.base.BaseFragment
+import com.sun.basic_japanese.constants.BasicJapaneseConstants
+import kotlinx.android.synthetic.main.fragment_alphabet.*
+
+class AlphabetFragment : BaseFragment(), AlphabetContract.View {
+    private var alphabetPresenter: AlphabetPresenter? = null
+    private var listener: OnFragmentInteractionListener? = null
+
+    override fun setPresenter(presenter: AlphabetContract.Presenter) {
+        alphabetPresenter = presenter as AlphabetPresenter
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
Dùng dấu "=" cho gọn nhé 
```suggestion
    ): View? = inflater.inflate(R.layout.fragment_alphabet, container, false)
```